### PR TITLE
Deploy to base

### PR DESCRIPTION
**Pull request by tuler**
Deploy contracts to [base network](https://www.base.org).

### CODE REVIEW

**Comment by guidanoli** [comment]
The `pnpm run prettier` command on `packages/contracts` is not working.

```
No parser could be inferred for file "...".
```

I've added the following line to the `.prettierrc.json` file, mimicking `rollups-contracts`, it seems to have fixed the problem for me.

```diff
diff --git a/packages/contracts/.prettierrc.json b/packages/contracts/.prettierrc.json
index 100e63c..cca0824 100644
--- a/packages/contracts/.prettierrc.json
+++ b/packages/contracts/.prettierrc.json
@@ -1,5 +1,6 @@
 {
     "tabWidth": 4,
+    "plugins": ["prettier-plugin-solidity"],
     "overrides": [
         {
             "files": "*.sol",
```

**Comment by guidanoli** [comment]
I was not able to figure out the `tsc` error when we migrated from `yarn` to `pnpm` though...

**Comment by guidanoli** [comment]
Setting `skipLibCheck` to `true` on `tsconfig.json` worked, but I'm not sure whether it's safe to go down this path.

**Comment by tuler** [comment]
> skipLibCheck

I think it's a fair compromise for now.

**Comment by guidanoli** [comment]
It appears that even big projects such as [OneKey](https://github.com/OneKeyHQ/app-monorepo) that use `viem@^2` use this configuration.

**Comment by guidanoli** [comment]
There are references to Yarn in the `README`

**Comment by tuler** [comment]
> It appears that even big projects that depend on `viem@^2` such as [OneKey](https://github.com/OneKeyHQ/app-monorepo) use this configuration.

It should definitely not be needed, but I would worry in this case

**Comment by guidanoli** [comment]
> It should definitely not be needed, but I would worry in this case

I agree. I wonder, though, if this issue can be fixed later.

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ And in your solidity contract import it as:
 
 ## Build
 
-To get a list of all available `yarn` targets run the following inside `packages/contracts`:
+To get a list of all available `pnpm` targets run the following inside `packages/contracts`:
 
-    % yarn run info
+    % pnpm run info
```
Comment (guidanoli) [inline README.md:53]: This command fails with the following error message.

```
Missing script: info
```

The equivalent to `yarn run info` seems to be just `pnpm run`. It prints:

```
Commands available via "pnpm run":
  build
    turbo run build
  dev
    turbo run dev --parallel
  lint
    turbo run lint
  format
    prettier --write "**/*.{ts,tsx,md}"
```

Comment (tuler) [reply README.md:53]: It must be run from inside `packages/contracts`, as stated in the README

Comment (guidanoli) [reply README.md:53]: Oops, you're right.

**Review by guidanoli — APPROVED** [review]
Optionally, I would sort entries in the `tsconfig.json`.

**Pull request merged**